jobs/JEVS_CAM_PLOTS

```diff
@@ -85,7 +86,7 @@ if  [ $MODELNAME = href ] ; then
   export DATA_IN=${DATA_IN:-$DATAROOT}
   export ush_dir=$USHevs/cam/ush_href_plot_py
 else
-  export COMIN=${COMIN:-$(compath.py -o $NET/$evs_ver)}
+  export COMIN=${COMIN:-$(compath.py $NET/$evs_ver)}
```

Comment:
Since the -o flag is not specified, this needs to change to 
```
export COMIN=${COMIN:-$(compath.py ${envir}/com/$NET/$evs_ver)}
```

Reply:
@ShelleyMelchior-NOAA Should I put the -o back in then, since this is a shared J-job?  I thought COMIN needs to not have -o.  Or go ahead and make the change you suggest?